src/forge/sync.py

```diff
@@ -0,0 +1,107 @@
+"""Rsync user content to EC2 instance."""
+import logging
+import os
+import subprocess
+import sys
+
+from . import DEFAULT_ARG_VALS, REQUIRED_ARGS
+from .destroy import destroy
+from .exceptions import ExitHandlerException
+from .parser import add_basic_args, add_general_args, add_env_args, add_action_args
+from .common import ec2_ip, key_file, get_ip, get_nlist, exit_callback
+
+logger = logging.getLogger(__name__)
+
+
+def cli_sync(subparsers):
+    """adds s3-sync parser to subparser
+
+    Parameters
+    ----------
+    subparsers : argparse.ArgumentParser
+        Argument parser for Forge.main
+    """
+    parser = subparsers.add_parser('sync', description='Rclone user content to EC2 instance')
+    add_basic_args(parser)
+
+    add_general_args(parser)
+    add_action_args(parser)
+    add_env_args(parser)
+
+    REQUIRED_ARGS['s3_sync'] = ['name',
+                              'service',
+                              'forge_env',
+                              'rsync_path']
+
+
+def s3_sync(config):
+    """rclones the file at rclone_path to the instance
+
+    Parameters
+    ----------
+    config : dict
+        Forge configuration data
+
+    Returns
+    -------
+    int
+        The status of the rsync commands
+    """
+    rval = 0
+
+    def _rclone(config, ip):
+        """performs an rclone to a given ip
+
+        Parameters
+        ----------
+        config : dict
+            Forge configuration data
+        ip : str
+            IP of the instance to rsync to
+        """
+        pem_secret = config['forge_pem_secret']
+        region = config['region']
+        profile = config.get('aws_profile')
+        rsync_loc = config.get('sync_path', config.get('app_dir'))
```

Comment:
I would change this variable to `sync_loc` or `rclone_loc` for consistency and to avoid confusion with the rsync command